ras/smtstate.py

```diff
@@ -67,3 +73,26 @@ def test(self):
                     self.log.info("SMT load is successful for SMT=%s" % j)
                 else:
                     self.fail("smt load failed")
+
+    def test_smt_state_switching(self):
+        """
+        Test to check the time taken for switching between different
+        SMT levels. The time taken must not exceed beyond 5 minutes.
+        """
+        process.system("ppc64_cpu --smt=on")
+
+        for i in [0, 1, 2, 4]:
+            if i == 0:
+                cmd_output = process.run(
+                    "/usr/bin/time -p ppc64_cpu --smt off", shell=True)
```

Comment:
is `/usr/bin/time` is available for no root user if not i recommend please use `sudo=True`

Reply:
I suppose, it is available, but _sudo=True_ better approach, just in case.

Reply:
Done